pages/main.js

```diff
@@ -633,7 +633,7 @@ var homepageInit = async function () {
 
     votesOutput += `
       <div class="votes-entry">
-        <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=${map.id}" target="_blank">
+        <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=${map.id}" target="_blank" class="nofade">
```

Comment:
Why not append this class to the container `div` instead? Seems cleaner than styling a link element, and the container already has a `class` property.

Reply:
This is because on pc that link element has the size 0xsomething and is placed at the bottom of the image. Setting the opacity of this overwrites the opacity of the image making it not start fading in until the image bottom enters the screen and that looks very bad.